Auth0.podspec

```diff
@@ -54,7 +54,7 @@ Pod::Spec.new do |s|
   s.social_media_url = 'https://twitter.com/auth0'
 
   s.ios.deployment_target = '9.0'
-  s.osx.deployment_target = '10.10'
+  s.osx.deployment_target = '10.11'
```

Comment:
current macOS 10.14, 10.10 was release October 16, 2014
Related to https://github.com/auth0/Auth0.swift/issues/269, this is a potential BC. I might think about a workaround instead.